src/chat/Chat.js

```diff
@@ -33,7 +33,7 @@ export default class Chat extends PureComponent {
 
   handleSend = () => {
     if (this.listComponent && this.listComponent.scrollToEnd !== undefined) {
-      this.listComponent.scrollToEnd();
+      setTimeout(this.listComponent.scrollToEnd, 300);
```

Comment:
This change is because when handleSend is triggered (for scrolling), the renderedMessages in MessageListContainer is not updated yet, hence it was not scrolling to the correct position.